express-post-json/index.js

```diff
@@ -0,0 +1,28 @@
+const express = require('express');
+const app = express();
+
+let nextId = 1;
+const grades = {};
+
+app.get('/api/grades', (req, res) => {
+  const array = [];
+  for (const key in grades) {
+    const objects = grades[key];
+    array.push(objects);
+  }
+  res.json(array);
+});
+
+app.use(express.json());
+
+app.post('/api/grades', (req, res) => {
+  grades[nextId] = req.body;
+  grades[nextId].id = nextId;
+  nextId++;
+  res.status(201).send(grades[nextId]);
```

Comment:
This part is confusing me. Wouldn't `grades[nextId]` be undefined after you incremented nextId?